apps/daimo-mobile/src/view/sheet/FarcasterBottomSheet.tsx

```diff
@@ -105,6 +105,7 @@ function FarcasterQRButton({ url }: { url: string }) {
         title="OPEN IN WARPCAST"
         onPress={openInWarpcast}
       />
+      <Spacer h={16} />
```

Comment:
might not be necessary anymore. FarcasterBottomSheet is now fixed height

Reply:
dropped